packages/shared/src/components/filters/TagsFilter.tsx

```diff
@@ -68,19 +87,42 @@ export default function TagsFilter({
       action: () => onUnblockTags({ tags: [tag] }),
     });
   };
+
   return (
     <div
       className="flex flex-col"
       aria-busy={isLoading}
       data-testid="tagsFilter"
     >
-      <SearchField
-        inputId="search-filters"
-        className="mx-6 mb-6"
-        ref={searchRef}
-        valueChanged={setQuery}
-      />
-      {query?.length > 0 ? (
+      <div
+        className={classNames(
+          containerClass[feedFilterModalType] ?? 'flex-col',
+          'flex px-6 pb-6',
+        )}
+      >
+        <SearchField
+          inputId="search-filters"
+          placeholder="Search"
+          className={classNames(searchFieldClass[feedFilterModalType], 'mb-6')}
```

Comment:
The `v3` value would be overriden, you would have to make it an optional value, else you will have to set something for every case.
```suggestion
          className={classNames(searchFieldClass[feedFilterModalType] || 'mb-6')}
```